README.md

```diff
@@ -42,9 +42,8 @@ container.openPath('local/path/to/game.html');
 ### Opening a Game Hosted at Another Domain
 
 ```javascript
-const container = new Container({
-  iframeSelector: '#game'
-});
+const iframe = document.querySelector('iframe');
```

Comment:
We should probably have both methods here, since it's going to allow for either a selector string or the HTML element itself.